app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewErrorView.kt

```diff
@@ -29,6 +29,7 @@ class LinkPreviewErrorView : LinearLayout {
 
         if (errorType === LinkPreviewErrorType.OFFLINE) {
             val message = (resources.getString(R.string.page_offline_notice_cannot_load_while_offline) +
+                    " " +
```

Comment:
Instead of a space, let's make this a newline character (`\n`)

Reply:
Surely! Will change it by the end of the day.

Reply:
No, not really, unless we want to turn it into two TextViews that appear next to each other, but that's not necessary for now.

Reply:
Done.